README.md

```diff
@@ -367,6 +367,18 @@ If unsure, please check what account you are using in the _Google Home_ app and
 
 If you can see your devices, and they all seem correct, but the alarms/timers sensors do not appear, or appear empty try restarting the Google Home device, Home Assistant, and reinstalling the integration.
 
+### Device offline or not compatible message (#402)
+
+If you get the following error for ***all*** of your devices:
```

Comment:
Based on the failed linting: https://github.com/leikoilja/ha-google-home/runs/4373080559?check_suite_focus=true

```suggestion
If you get the following error for **_all_** of your devices:
```